yarn.lock

```diff
@@ -22183,6 +22582,13 @@ onetime@^5.1.0:
   dependencies:
     mimic-fn "^2.1.0"
 
+onetime@^5.1.2:
+  version "5.1.2"
+  resolved "https://registry.yarnpkg.com/onetime/-/onetime-5.1.2.tgz#d0e96ebb56b07476df1dd9c4806e5237985ca45e"
+  integrity sha512-kbpaSSGJTWdAY5KPVeMOKXSrPtr8C8C7wodJbcsd51jRnmD+GZu8Y0VoU6Dm5Z4vWr0Ig/1NKuWRKf7j5aaYSg==
+  dependencies:
+    mimic-fn "^2.1.0"
```

Comment:
It looks like `onetime@^5.1.2` and `onetime@^5.1.0` above can be collapsed into a single dependency.

Reply:
Hmm, not sure, it's `onetime` not `twotimes`! :slightly_smiling_face: